cmd/minishift/cmd/start.go

```diff
@@ -48,6 +48,7 @@ import (
 
 	"github.com/minishift/minishift/pkg/util"
 	"github.com/minishift/minishift/pkg/util/os/atexit"
+	"github.com/minishift/minishift/pkg/util/progress"
```

Comment:
The package name `progress` should be changed to `progressDots` or something better.

Reply:
package names do not have casing progressdots is possible, but not sure if this really adds value.

Reply:
If progressDots is used as a package name, it clashes with:
```
progressDots := progress.New()
```

and therefore would need an override from the import to use
```
    progressdots"github.com/minishift/minishift/pkg/util/progressDots"
```

`progressdots` it is... changing and merging...